priv/repo/seeds.exs

```diff
@@ -1,147 +1,2 @@
-require Logger
-# Script for populating the database. You can run it as:
-#
-# mix run priv/repo/seeds.exs
-#
-# The seeds.exs script will also run
-# when you execute the following command
-# to setup the database:
-#
-# mix ecto.setup
-defmodule Auth.Seeds do
```

Comment:
The bulk of this code moved to [`init.ex`](https://github.com/dwyl/auth/pull/175/files#diff-d106602d01c212adbdc5cc818402766b0907b01a9a0da2bfea97e97e4e8d912b) above. ⬆️